src/tools/tidy/Cargo.toml

```diff
@@ -12,6 +12,7 @@ miropt-test-tools = { path = "../miropt-test-tools" }
 walkdir = "2"
 ignore = "0.4.18"
 semver = "1.0"
+serde = { version = "1.0.125", features = [ "derive" ] }
```

Comment:
Question: should this be gated behind like a `build-metrics` cargo feature? I ask because `tidy` is built very often (many people have pre-push hooks that will run tidy) so build time is important.

Reply:
ye I can totally do that, that sounds like a good idea to me.